src/components/checkboxSet/CheckboxSet.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react'
+import { AppContext } from '../appContext'
+import Checkbox from '../checkbox'
+import constants from '../../constants'
+import { List, ListItem } from './CheckboxSet.styles'
+import { CheckboxSetProps } from './CheckboxSet.types'
+
+const CheckboxSet = ({ filterName, sort = null }: CheckboxSetProps) => {
+  const appContext = useContext(AppContext)
```

Comment:
Could you please destructure `appContext` here? 🙂

Reply:
Thank you. I have de-structured state and actions from appContext, is that good enough?

Reply:
Yup, thanks!